### DISCUSSION TITLE
Ability to save server/key per Remote Desktop instead of global setting in client (multiple servers)

### DISCUSSION THREAD

**Discussion by doslagerRubrik** [rustdesk/rustdesk#4387] [Feature Request]
Hi, 

I have my own private RustDesk server setup for my internal hosts. Sometimes I have to connect to third party systems for remote support and i tell them to use RustDesk portable for a one-off remote session. 

To connect, I have to delete the relay server and encryption key from my client. For one-off situations, it doesn't make sense (to me) to have them go through the server/Key setup. And, I would rather not have my key exposed to one-off third-party situations. 

It would be great to be able to set that on a per connection setting rather than global. Hopefully this makes sense and I'm not missing something obvious on my end. 

Thank you! Great product!

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/2118
https://github.com/rustdesk/rustdesk/discussions/1258 
https://github.com/rustdesk/rustdesk/discussions/1873

**Comment by SergeyMy** [comment]
> To connect, I have to delete the relay server and encryption key from my client. 
>
**sudo -u support_users rustdesk**
settings are stored in the user profile :).
